201-vmss-slb-ilb/azuredeploy.json

```diff
@@ -0,0 +1,393 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "type": "string",
+      "defaultValue": "changeme",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "sourceImageVhdUri": {
+      "type": "string",
+      "defaultValue": "changeme",
+      "metadata": {
+        "description": "The source of the blob storage containing the custom image."
+      }
+    },
+    "vmssName": {
+      "type": "string",
+      "defaultValue": "VmssWithSlbIlb",
```

Comment:
Since this is globally unique remove the defaultValue or provide a valid one using uniqueString()